css/components/slider.css

```diff
@@ -0,0 +1,20 @@
+.slider-container {
+  padding: 2rem 0;
+  width: 100%;
+}
+
+input[type="range"] {
+  accent-color: hsl(var(--accent));
+  display: block;
+  outline: 1px solid transparent;
+  outline-offset: 2px;
```

Comment:
I think this set of rules can be applied to all the buttons and inputs in the base file as a reset, since it is needed everywhere for the buttons and inputs.

Reply:
will rrefactor later